markdown/usage/troubleshooting.md

```diff
@@ -437,7 +437,9 @@ To fix this you need to change the default memory requirements for the process t
 
 For example, let's say it's the `markduplicates` process that is running out of memory (as displayed on the Nextflow running display).
 
-- First we need to check to see what default memory value we have. Go to the main code of the pipeline by going to the corresponding pipeline GitHub repository and open the `main.nf` file. Search in your browser for `process markduplicates`.
+- First we need to check to see what default memory value we have. For this you need to go into the pipeline code locally in your local copy (typically: `~/.nextflow/assets/nf-core/<pipeline>/`, or the GitHub repository (`https://github.com/nf-core/<pipeline>`)
+    - DSL2 Pipelines (i.e., has a `modules/` directory): Go to the file here `modules/nf-core/<tool_name>/main.nf`, and check the `label` line.
+    - DSL1 Pipelines (i.e., does not have a `modules/` directory): Go to the main code of the pipeline by going to the corresponding pipeline GitHub repository and open the `main.nf` file. Search in your browser for `process markduplicates`.
```

Comment:
```suggestion
    - If the pipeline has a `modules/` directory, i.e. is a DSL2 pipeline: Go to the file here `modules/nf-core/<tool_name>/main.nf`, and check the `label` line.
    - If the pipeline doesn't have a `modules/` directory, i.e. is a DSL1 pipelines. Go to the main code of the pipeline by going to the corresponding pipeline GitHub repository and open the `main.nf` file. Search for `process markduplicates`.
```

Reply:
I think by switching the order and putting the jargon after the explanation, it should be a bit less scary for newcomers.

Reply:
Good call